packages/material/scss/pivotgrid/_theme.scss

```diff
@@ -0,0 +1,142 @@
+@include exports('pivotgrid/theme') {
+    $pivotgrid-alt-bg: darken($bg-color, 4%) !default;
+    $pivotgrid-alt-color: darken($bg-color, 82%) !default;
+    $pivotgrid-chrome-border: $border-color !default;
+    $pivotgrid-container-bg: $bg-color !default;
+
+    $pivotgrid-button-bg: darken($bg-color, 7%) !default;
+    $pivotgrid-button-hover-bg: darken($bg-color, 57%) !default;
+    $pivotgrid-button-active-bg: darken($bg-color, 54%) !default;
+
+    $pivotgrid-remove-color: $pivotgrid-button-bg !default;
+    $pivotgrid-remove-bg: $pivotgrid-button-active-bg !default;
+    $pivotgrid-remove-hover-color: $pivotgrid-button-active-bg !default;
+    $pivotgrid-remove-hover-bg: $bg-color !default;
+
+    .k-alt,
+    .k-fieldselector,
+    .k-pivot-layout .k-grid-footer,
+    .k-pivot-layout > tbody > tr:first-child > td:first-child {
+        background-color: $pivotgrid-alt-bg;
+    }
+
+    .k-pivot-toolbar,
+    .k-header.k-alt,
+    .k-pivot-layout .k-grid.k-alt,
+    .k-pivot-rowheaders .k-alt .k-alt,
+    .k-fieldselector .k-list-container {
+        background-color: $pivotgrid-container-bg;
+    }
+
+    .k-pivot-toolbar,
+    .k-pivot-table,
+    .k-fieldselector .k-list-container,
+    .k-fieldselector .k-columns > div,
+    .k-pivot-rowheaders > .k-grid td:first-child,
+    .k-pivot-table .k-grid-header .k-header.k-first {
+        border-color: $pivotgrid-chrome-border;
+    }
+
+    .k-pivot-rowheaders > .k-grid td,
+    .k-pivot-rowheaders > .k-grid td:first-child {
+        border-bottom-color: transparent;
+        border-left-width: 1px;
+    }
+
+    .k-pivot-layout .k-grid .k-alt{
+        border-left-width: 1px;
+
+        + .k-alt {
+            border-left-color: transparent;
+        }
+    }
+
+    .k-fieldselector p,
+    .k-pivot-layout .k-button {
+        text-transform: none;
+    }
+
+    .k-pivot-layout .k-grid-footer,
+    .k-pivot-layout .k-grid .k-alt {
+        color: $pivotgrid-alt-color;
+        font-weight: bold;
+    }
+
+    .k-pivot-toolbar .k-button,
+    .k-fieldselector .k-list li.k-item {
+        @include appearance( button );
+        background-clip: padding-box;
+        background-color: $pivotgrid-button-bg;
+        box-shadow: unset;
+
+        .k-field-actions{
```

Comment:
Keep attention to whitespaces - we have whitespace after selector and curly bracket